lib/CrossTU/CrossTranslationUnit.cpp

```diff
@@ -240,12 +240,34 @@ CrossTranslationUnitContext::getCrossTUDefinition(const FunctionDecl *FD,
   }
   const auto& LangTo = Context.getLangOpts();
   const auto& LangFrom = Unit->getASTContext().getLangOpts();
```

Comment:
(I do not like `auto` here and the & is at wrong place (but not part of the change).)

Reply:
Yes I agree, but once we will port back https://reviews.llvm.org/D55134 where this part will be updated.